s3tests_boto3/functional/test_s3.py

```diff
@@ -152,6 +152,39 @@ def test_bucket_list_many():
     eq(response['IsTruncated'], False)
     eq(keys, ['foo'])
 
+def test_bucket_listv2_many():
```

Comment:
@albIN7 this test is missing tags.

Reply:
This test still needs all the tags `test_bucket_list_many` has.

Reply:
Added.